### ISSUE TITLE
Truncated content - maybe branch overlap problem?

### ISSUE THREAD

**Issue by working-name** [vipzhicheng/logseq-plugin-mark-map#60]
Hi,

It seems a branch's content might get truncated because it tries to fit the other branch that's around it? Not sure, but see screenshots :)

![y u eatin](https://user-images.githubusercontent.com/16505052/227791252-c7d8b5a1-1e4b-4f53-8350-da9061be03c3.png)
![he ate it](https://user-images.githubusercontent.com/16505052/227791263-674c0c16-675e-4344-b729-9e038fd9f4b6.png)

Please let me know if there's anything you need from me.

Thanks!

**Comment by working-name** [comment]
No, I don't think it has anything to do with fitting other branches.
![Screenshot 2023-03-26 115419](https://user-images.githubusercontent.com/16505052/227791469-c4e917f8-59ca-40e9-9145-4927e8cf86e0.png)

**Comment by vipzhicheng** [comment]
I can not reproduct this issue, please provide a pure `.md` file for me. Thx.
